### DISCUSSION TITLE
How do I host my own rigtools link?

### DISCUSSION THREAD

**Discussion by thatboyo** [3kh0/ext-remover#1397] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE]
I've followed the directions, ran the commands, pasted the {"updaterurl:localhost:8080 "} thingy, but i can tfind my websocket or local host, what do i do from here?

**Comment by riseofthycyberdemon** [comment]
`localhost` is your computer trying to make the request itself (Which is why it's called `localhost`.)
If you want to have a specialized link, like `rig.cssd.store`, you are going to have to do your research because that is beyond the scope of my knowledge. Hope this helps enough.

**Reply by thatboyo** [reply]
so do i just put "devtools://devtools/bundled/devtools_app.html?experiments=true&ws=localhost:8080" ?

**Reply by immlad** [reply]
https://rigtools.playcode.io/

**Reply by immlad** [reply]
yeah but im just commenting to see if it could help some people

**Comment by Charlieis1nsan3** [comment]
No shot bro said "How Do I HOST my own RigTools link."  you don't host RigTools. like Im not tryna be a dick head but you don't "host" RigTools.

**Reply by thatboyo** [reply]
due i was just following this terminology 
![Uploading Screenshot 2024-12-18 7.34.59 AM.png…]()

**Reply by thatboyo** [reply]
im trying to create a new link, and the 3kh0 post said "self host"

**Reply by thatboyo** [reply]
please someone help

**Comment by S-PScripts** [comment]
use this to comment instead of https://github.com/3kh0/ext-remover/discussions/1424

**Reply by immlad** [reply]
alr
